project/Paradox.scala

```diff
@@ -114,6 +111,5 @@ object Paradox {
     sourceGeneratorSettings ++
     Seq(
       Compile / paradox / name := "Pekko",
-      resolvers += Resolver.jcenterRepo,
```

Comment:
because it's now published to maven center?

Reply:
jcenter is no longer supported - it was important years ago - but not now

https://blog.gradle.org/jcenter-shutdown